pkg/vm/airdrop.go

```diff
@@ -13,6 +13,12 @@ import (
 	"github.com/ethereum/go-ethereum/common"
 )
 
+const (
```

Comment:
not sure is this should be module level or function level consts

Reply:
I needed to do this to mock the prompt in the test.